cristin-person/src/main/java/no/unit/nva/cristin/person/employment/query/QueryPersonEmploymentClient.java

```diff
@@ -1,28 +1,30 @@
 package no.unit.nva.cristin.person.employment.query;
 
+import no.unit.nva.cristin.common.client.ApiClient;
+import no.unit.nva.cristin.model.SearchResponse;
+import no.unit.nva.cristin.person.model.cristin.CristinPersonEmployment;
+import nva.commons.apigateway.exceptions.ApiGatewayException;
+import nva.commons.apigateway.exceptions.BadRequestException;
+import nva.commons.core.paths.UriWrapper;
+
+import java.net.HttpURLConnection;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpResponse;
+import java.util.List;
+
 import static java.util.Arrays.asList;
 import static no.unit.nva.cristin.model.Constants.BASE_PATH;
 import static no.unit.nva.cristin.model.Constants.CRISTIN_API_URL;
 import static no.unit.nva.cristin.model.Constants.DOMAIN_NAME;
 import static no.unit.nva.cristin.model.Constants.HTTPS;
 import static no.unit.nva.cristin.model.Constants.PERSON_PATH;
 import static no.unit.nva.cristin.model.Constants.PERSON_PATH_NVA;
-import static no.unit.nva.cristin.person.employment.Constants.EMPLOYMENT_PATH;
-import static no.unit.nva.cristin.person.employment.Constants.EMPLOYMENT_PATH_CRISTIN;
-import java.net.HttpURLConnection;
-import java.net.URI;
-import java.net.http.HttpClient;
-import java.net.http.HttpResponse;
-import java.util.List;
-import no.unit.nva.cristin.common.client.ApiClient;
-import no.unit.nva.cristin.model.SearchResponse;
-import no.unit.nva.cristin.person.model.cristin.CristinPersonEmployment;
-import nva.commons.apigateway.exceptions.ApiGatewayException;
-import nva.commons.apigateway.exceptions.BadRequestException;
-import nva.commons.core.paths.UriWrapper;
 
 public class QueryPersonEmploymentClient extends ApiClient {
 
+    public static final String EMPLOYMENT_PATH_CRISTIN = "affiliations";
```

Comment:
See class `Constants` in package `employment`

Reply:
Changes reverted